nexus/bin/qmca

```diff
@@ -2677,6 +2683,18 @@ QMCA examples:
             #end for
         #end for
         opt = self.options
+        if opt.average and opt.save_average:
+            for fmap in self.file_map:
+                ndata = len(fmap.data['LocalEnergy'])
+                data = [arange(ndata)]
+                keys = 'index'
+                fmt = ['%i']
+                for key in fmap.data.keys():
+                    data.append(fmap.data[key])
+                    keys+='     {}'.format(key)
+                    fmt.append('%.10e')
```

Comment:
Format control expressions to match surrounding code, i.e. use "#end if" and "#end for".